test/logflare_web/live/admin/partner_live_test.exs

```diff
@@ -0,0 +1,60 @@
+defmodule LogflareWeb.Admin.PartnerLiveTest do
+  use LogflareWeb.ConnCase
+  import Phoenix.ConnTest
+  import Phoenix.LiveViewTest
+  @endpoint LogflareWeb.Endpoint
+
+  describe "PartnerLive" do
+    setup %{conn: conn} do
+      user = insert(:user, admin: true)
+      insert(:plan)
+      conn = login_user(conn, user)
+      {:ok, conn: conn}
+    end
+
+    test "view with no partners renders", %{conn: conn} do
+      {:ok, _view, html} = live(conn, "/admin/partner")
+
+      assert html =~ "No partners created yet"
+    end
+
+    test "view with partners renders", %{conn: conn} do
+      partners = insert_list(5, :partner)
+      {:ok, _view, html} = live(conn, "/admin/partner")
+
+      Enum.each(partners, fn %{name: name, token: token, auth_token: auth_token} ->
+        assert html =~ name
+        assert html =~ token
+        assert html =~ auth_token
```

Comment:
don't need to reveal the auth tokens

Reply:
We need for now, Partners will be created be us and we will share credentials with said partners (e.g. Chase will send Platform A their credentials to access the Partner API and start using it)

I don't think this is something we require self-service for now. When the time comes to provide self-service we have the foundations to do that by reusing the ideas of this page with some alterations.